docs/writing-tests/general-guidelines.md

```diff
@@ -117,6 +117,31 @@ This is not, however, to discourage testing of edge cases or
 interactions between multiple features; such tests are an essential
 part of ensuring interoperability of the web platform.
 
+Tests should pass when the feature under test exposes the expected behavior,
+and they should fail when the feature under test is not implemented or is
+implemented incorrectly. Tests should not rely on unrelated features if doing
+so causes failures in the latest stable release of [Apple
+Safari][apple-safari], [Google Chrome][google-chrome], [Microsoft
+Edge][micosoft-edge], or [Mozilla Firefox][mozilla-firefox]. They should,
```

Comment:
```suggestion
Safari][apple-safari], [Google Chrome][google-chrome],
or [Mozilla Firefox][mozilla-firefox]. They should,
```
…since below we say *“only use language features that have been supported by **each of the major browser engines above** for over a year”*, and Edge is no longer among the set of browsers that originate their own engines.